src/planet-a/IcoSphereTree.h

```diff
@@ -90,33 +166,36 @@ class IcoSphereTree
 
     void initialize(float radius);
 
+    trindex_t triangle_count() { return m_triangles.size(); }
+
     /**
      * Get triangle from vector of triangles
      * be careful of reallocation!
-     * @param t [in] Index to triangle
+     * @param triInd [in] Index to triangle
      * @return Pointer to triangle
      */
-    SubTriangle& get_triangle(trindex_t t)
-    {
-        return m_triangles[t];
-    }
-
-    trindex_t triangle_count()
+    SubTriangle& get_triangle(trindex_t triInd)
```

Comment:
technically this is constexpr as well

Reply:
The [] operator of std::vector is not constexpr